python/lsst/daf/persistence/policy.py

```diff
@@ -250,6 +210,24 @@ def __contains__(self, key):
                 return False
         return keys[-1] in d
 
+    @staticmethod
+    def defaultPolicyFile(productName, filePath, repos=None):
+        """Get the path to a default policy file
+
+        @param productName (string) The name of the product that the default policy is installed as part of
+        @param filepath (string) The relative pathname to the policy file.  
+        @param repos (string) the subdirectory with the product's install directory where policy files are 
+                              stored. If None (default), the filepath argument is relative to the installation
+                              directory.
```

Comment:
Ok, I'm actually having a hard time working out how `filepath` and `repos` relate to each other from this description without looking at the code. Is the actual file in `productDir/repos/filepath`? Is it also `productDir/filepath` if `repos` is `None`? Does `filepath` include the actual file itself or just the path to the file? If it includes the file does it include the extension? All these questions were in my head as I read this description.